index.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: index
+---
+
+## Key concepts
+
+> Auditree aims to be opinionated in how you work, but flexible in what work
+> you do.
+
+Everything revolves around *evidence*. Simple programs called *fetchers*
+retrieve evidence from sources (API's, URL's, command line invocations) and
+store that evidence in a *locker*. This is a git repository (so efficient,
+tamper evident and versioned for storing evidence) with some additional
+controls & metadata inserted by the supporting framework. Evidence can then be
+verified by *checks*; simple unit tests against your evidence.
+
+Your checks can be use to assert an operational posture, implement secondary
+controls or be a primary control in your compliance posture.
+
+Notification of check status can be sent to a issues, Slack, PagerDuty, or
```

Comment:
drop the `a` between `sent to` and `issues`.

Reply:
Maybe say Github issues.

Reply:
I wanted to be github agnostic - in theory we could add jira etc

Reply:
though in the other bit I say github issue...